samples/contrib/versioned-pipeline-ci-samples/mnist-ci-sample/README.md

```diff
@@ -0,0 +1,31 @@
+# Mnist Continuous Integration(CI) Pipeline
+
+## ## Overview
+
+This sample uses cloud build to implement the continuous integration process of a basic machine learning pipeline that trains and visualizes model in tensorboard. Once all set up, you can push your code to github repo, then the build process in cloud build will be triggered automatically, then a run will be created in kubeflow pipeline. You can view your pipeline and the run in kubeflow pipelines. 
+
+We use *Kubeflow Pipeline(KFP) SDK** to interact with kubeflow pipeline to create a new version and a run in this sample.
+
+## What you can learn in this sample
+* CI process of a simple but general ML pipeline.
+* Launch a tensorboard as one pipeline step
+* Data passing between steps
+
+
+## What needs to be done before run
+* Create a secret following the troubleshooting parts in [https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize]()
```

Comment:
Maybe put the link in the brackets to make it work?

Reply:
Yes, Thanks.